monday/tests/test_board_resource.py

```diff
@@ -39,6 +39,14 @@ def test_get_boards_by_id_query(self):
     def test_get_board_items_query(self):
         query = get_board_items_query(board_id=self.board_id)
         self.assertIn(str(self.board_id), query)
+        items_line = f'items()'
```

Comment:
Would prefer that this be its own test i.e. with and w/o pagination given that both are valid use cases, even if the second test is repetitive.